dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCClient.scala

```diff
@@ -883,7 +883,8 @@ case class BinaryOutcomeDLCClient(
     */
   def executeRemoteUnilateralDLC(
       dlcSetup: SetupDLC,
-      publishedCET: Transaction): Future[UnilateralDLCOutcome] = {
+      publishedCET: Transaction,
+      sweepPrivKey: ECPrivateKey): Future[UnilateralDLCOutcome] = {
```

Comment:
Would it be better to pass in a `SpendingInfo` so we can sweep to any addr we want?

Reply:
at some point I think that would be nice, would prefer to do it on another PR though as this one is slowly growing in scope :P